tests/kafkatest/services/verifiable_consumer.py

```diff
@@ -398,8 +398,9 @@ def start_cmd(self, node):
         # The two implementations use slightly different configuration, hence these arguments are conditional.
         #
         # See the Java class/method VerifiableConsumer.createFromArgs() for how the command line arguments are
-        # parsed and used as configuration in the runner.
-        if node.version >= V_3_7_0 and self.is_consumer_group_protocol_enabled():
+        # parsed and used as configuration in the runner. Note: even though support for the two group protocols was
+        # introduced in 3.7.0, support for the --group-protocol option was not added until later.
+        if node.version > V_3_7_1 and self.group_protocol:
```

Comment:
with this change we would no longer be running the sys tests with CONSUMER for nodes on v3.7.0 I expect, right? Is that the intention? Sorry if I missing something here

Reply:
Correct.

Reply:
but is that right? this means that running system tests with 3.7.0 will never exert the new consumer/protocol, and it doesn't feel right because 3.7.0 does include the new consumer/protocol. Am I missing something?

Reply:
Unfortunately, the changes to `VerifiableConsumer.java` to add the group protocol argument (`--group-protocol`) were not added until 3.7.2. The result being that when the system tests that validate client compatibility attempt to test against 3.7.0 or 3.7.1 clients, an exception is thrown because the clients don't know about that command line option.

To fix this it seems like we'd have to go back to the 3.7.0 and 3.7.1 versions and retroactively add support for the `--group-protocol`. That would would then generate a new version such as 3.7.0.x and 3.7.1.x. But at that point the system tests would still fail until an official release of 3.7.0.1 and 3.7.1.1 are made and the tests are updated to specifically use those versions.

I'm not sure what we can do in 4.0 to fix this 🤔

Reply:
I see you were able to revert that, nice. Could you share a run of the sys tests with this latest changes to see where we stand? Thanks!

Reply:
Sorry for the late reply. System tests have been blocked by an environmental issue in our build environment. I believe this is now fixed, so I will start a test.